### ISSUE TITLE
the output of visual hidden between ViT-B/32 and RN50 doesn't match

### ISSUE THREAD

**Issue by cheetah003** [ArrowLuo/CLIP4Clip#16]
Hi ArrowLuo,
When I change the pretrained ViT-B/32 to RN50,, I got an error in function encode_image(module_clip.py).
The visual hidden output (x) dim of  ViT-B/32 is [768, 512],
while the visual hidden output (x) dim of RN50 is [768, 1024].
Do you know how to solve this problem? Thanks.

**Comment by ArrowLuo** [comment]
As I know, the output dim is set according to the pretrained state_dict. i.e., [embed_dim](https://github.com/openai/CLIP/blob/main/clip/model.py#L413). Thus, the text hidden output is also mapped to 1024.

**Comment by cheetah003** [comment]
Hi ArrowLuo,
Thanks for your quick response！I replaced the clip's text encoder with a chinese text encoder before,so I didn't notice that clip's text encode is also mapped to 1024. I found it is true after use the clip's text encoder again,
